docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,169 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
```

Comment:
We should have a paragraph or two here explaining what is this about and what are these animations for